### PR TITLE
Contributing layer requirements

### PR DESCRIPTION
**Pull request by MattF-NSIDC**

### CODE REVIEW

File: doc/CONTRIBUTING.md [hunk 1]
```diff
@@ -269,20 +269,33 @@ assistance or additional information. If you're not comfortable with GitHub,
 you can always [email the QGreenland team](mailto:qgreenland.info@gmail.com).
 
 
-# Layer Requirements
+### Layer Requirements
 
-Data must be:
+In order for a new dataset to be added to QGreenland, we strongly encourage
+public archival with OGC-compliant metadata. If data is not publicly archived
+or stored in a weird format, maintenance of that layer takes an order of
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:276]: 'weird' -> 'non-standard'?

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -269,20 +269,33 @@ assistance or additional information. If you're not comfortable with GitHub,
...
-* Subset to one of the defined layer boundaries in `config/project.yml`.
+* Is subset to one of the defined layer boundaries in `config/project.yml`.
   Existing layer tasks can do this for vector or raster data.
 
-* In the correct format. As of this writing, our current convention is to store
-  raster data in GeoTIFF (`.tif`) format, and vector data in GeoPackage
-  (`.gpkg`) format.
+* For raster data:
+  * In GeoTIFF (`.tif`) format.
+  * Includes overviews, for raster data. This improves QGIS performance.
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:293]: I think we currently have some one-off raster pipelines that do not generate overviews...maybe we should make a card to double-check that if we don't already.

Comment (trey-stafford) [reply doc/CONTRIBUTING.md:293]: Same w/ the next line about DEFLATE compression.

File: doc/CONTRIBUTING.md [hunk 3]
```diff
@@ -269,20 +269,33 @@ assistance or additional information. If you're not comfortable with GitHub,
...
+  * Is losslessly compressed using the DEFLATE algorithm.
+
+* For vector data:
+  * In GeoPackage (`.gpkg`) format.
+  * Uses the `label` attribute name for pre-calculated labels.
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:298]: perhaps expand this a bit: '... for pre-calculated labels when using one of the `named_*.qml` styles'

**Review by trey-stafford — APPROVED** [review]

File: doc/CONTRIBUTING.md (hunk 3 above)
Comment (MattF-NSIDC) [reply doc/CONTRIBUTING.md]: I think we should actually standardize around `label` field for even generic styles like `labeled_point.qml` (I think that's what it's called)

**Pull request merged**